connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/model/user_registry/EmbeddedExternalId.java

```diff
@@ -0,0 +1,14 @@
+
+package it.pagopa.selfcare.dashboard.connector.rest.model.user_registry;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
@KevinSi96 use @Data instead of @Getter and @Setter so Lombok will create the toString() method too